peer.go

```diff
@@ -276,6 +276,33 @@ func (p *peer) Start() error {
 	return nil
 }
 
+// getEdgeInfo returns the channel edge info.
```

Comment:
The commit title needs to be reworked a bit, how about: `lnd: hook up encrypted onion error scaffolding`.